Makefile

```diff
@@ -177,7 +177,9 @@ wasm-test: wasm-lib-test wasm-rego-test
 
 .PHONY: wasm-lib-build
 wasm-lib-build:
-ifeq ($(DOCKER_RUNNING), 1)
+ifeq ($(WASM_ENABLED), 0)
+	@echo "WASM_ENABLED=0. Skipping OPA-WASM library build."
```

Comment:
I'd rather not change these. Currently, I'm having trouble with _building_ with WASM_ENABLED=1, but these targets, since they run in docker, still work fine.

Reply:
I guess it depends on what the purpose of this flag should be. Setting WASM_ENABLED=0 to skip waiting for WASM tests when making e.g. doc changes would make sense to me. But you think this should only skip the "WASM WASM" tests? 😃

Reply:
> Setting WASM_ENABLED=0 to skip waiting for WASM tests when making e.g. doc changes would make sense to me.

If you want to avoid checking stuff for your docs change, I'd think that should be a matter of make targets, not env vars. Basically, if you don't want to run wasm stuff, don't `make wasm-lib-build` 😉 
What do you think?

Reply:
Given how these tests already are conditional on `DOCKER_RUNNING` (and that's what I've used to skip these tests previously) it doesn't seem like such a stretch. I never `make wasm-lib-build` directly, just `make test` .. guess I'll need just need to reprogram myself to change that to run `make go-test` instead.